src/set_theory/ordinal_arithmetic.lean

```diff
@@ -863,6 +863,54 @@ end
 theorem dvd_iff_mod_eq_zero {a b : ordinal} : b ∣ a ↔ a % b = 0 :=
 ⟨mod_eq_zero_of_dvd, dvd_of_mod_eq_zero⟩
 
+/-! ### Families of ordinals -/
+
+/-- Converts a family indexed by a `Type u` to one indexed by an `ordinal.{u}` using a specified
+well-ordering. -/
+@[simp]
```

Comment:
Is there a reason to mark all these defs `@[simp]`? It's not necessarily wrong, but I don't quite see what the API here is. Which of these defs/lemmas are implementation details, and which do we expect the end user to see and use? 

Actually, it might be nice to write a sentence or two explaining the API in the section comment a few lines above.

Reply:
I marked them as `simp` for pragmatic reasons: before I changed `simp` to `simp only`, marking these definitions as `@[simp]` allowed a few of the theorems I proved to be almost instant. I'll admit to not being very familiar with `simp`, so if this was wrong I can't tell.

The important part of this API is that it allows converting `(ι : Type u) → β` indexed families into `Π b < o, β` families and viceversa, which describe the same elements and thus should satisfy essentially the same results. As such, results about these different kinds of families are often easier to prove the second time around. Exactly how the conversion is done isn't relevant at all.

Reply:
I've been told what `@[simp]` does and I realized it was definitely wrong to mark these defs as `@[simp]`. I did mark the theorems, though.